### PR TITLE
integration docs beta 1/ replicate all integration pages from mkt site to beta docs

### PR DESCRIPTION
**Pull request by yuhan**
## Summary & Motivation
copy over from https://dagster.io/integrations (note: this content is up-to-date per PR stack https://github.com/dagster-io/dagster-website/pull/1280 few weeks ago)

this PR made the following changes:
1. update title to "Dagster & <name>"
2. add `sidebar_label: <name>" so it won't show a wall of "Dagster &" on left nav
3. fix all vale errors and warnings, including a lot of vale accept additions
4. rename files from `dagster-<name>.mdx` to just `<name>.md`

Next steps in later stack:
- move code to python files
- improve navigation: Index page and/or left nav to bucket integrations into categories, differentiate community owned

Later steps:
- improve doc content page one by one (e.g. template guides, reuse the good ones from current docs site)

**Open discussion:** Figure out the relationship between docs and marketing site regarding integrations. 
  * Option 1: no dagster.io/integrations and redirect that to docs.dagster.io/integrations
    * Yuhan's pick: I'm actually leaning towards this to completely consolidate all integration contents into the docs site for simplicity and ease of navigation so there won't be two similar content on two different sites, but I'd need to consult the SEO implication in this option.
  * Option 2: keep both dagster.io/integrations and docs.dagster.io/integrations; no code in marketing site, only for SEO purpose; docs pages focus on more technical guides/references.

## How I Tested These Changes
**see in preview: https://dagster-docs-beta-211qncb7r-elementl.vercel.app/integrations**

## Changelog
`NOCHANGELOG`

### CODE REVIEW

**Comment by yuhan** [comment]
* **#24330** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24330?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yuhan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by cmpadden** [comment]
This looks nice, @yuhan , some observations and questions:

- How will this interface with the full integration guides?
- We may want to group cloud provider integrations in sidebar, for example, `AWS > S3, ECR, ECS, etc`
- Having the integrations tab makes this appear as though it's an exhaustive list, we will want to ensure an entry exists for each integration
- We will want a landing index page on `/integrations`, maybe with tiles / icons
- We should link to integration API docs

File: docs/docs-beta/docs/getting-started/installation.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Installing Dagster"
-description: "Learn how to install Dagster"
+title: 'Installing Dagster'
```
Comment (PedramNavid) [inline docs/docs-beta/docs/getting-started/installation.md:2]: we can probably just get rid of all quotes

**Review by PedramNavid — COMMENTED** [review]
I am strongly in favor of Option 1. If we need to, we can improve the SEO of the docs site too.

**Comment by yuhan** [comment]
thanks @cmpadden i added a clarifying point in the summary and this PR stack is just the first steps to move what we currently have:
> Later steps:
> - improve doc content page one by one (e.g. template guides, reuse the good ones from current docs site)

i’m imagining that this^ work could be distributed once the docs team is less busy with core docs (i’m mostly trying to help pull work forward as i have some free cycles now).

- How will this interface with the full integration guides?
  * I feel the important ones would have their own full integration guides (like dbt), and the long tails ones would just have one pager guide.
  * One pager format is TBD too, and we may want to template them differently based on their category/pattern.
- We may want to group cloud provider integrations in sidebar, for example, AWS > S3, ECR, ECS, etc
  * Great idea. Will do a nav rework in a later stack.
- Having the integrations tab makes this appear as though it's an exhaustive list, we will want to ensure an entry exists for each integration
  * Good thought! Could maybe add some test to ensure all python modules in the `libraries` dir are included here. Need to think more about it.
- We will want a landing index page on /integrations, maybe with tiles / icons
  * I'm in favor of Option 1 and in that case, we'd migrate the landing page (https://dagster.io/integrations) over too
- We should link to integration API docs
  * Yes! Will do it in a later PR

**Review by cmpadden — APPROVED** [review]
Thanks Yuhan, approving with the expectation that future stacks will address the points in my comment.

I can do a fast follow-up on removing the quotes as Pedram mentioned.

File: docs/docs-beta/docs/getting-started/installation.md (hunk 1 above)
Comment (cmpadden) [reply docs/docs-beta/docs/getting-started/installation.md]: @yuhan here's a fast-follow to remove quotes for integration front-matter.
https://github.com/dagster-io/dagster/pull/24404

**Comment by yuhan** [comment]
* **#24330** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24330?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#24391](https://github.com/dagster-io/dagster/pull/24391) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24391?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#24404](https://github.com/dagster-io/dagster/pull/24404) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24404?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yuhan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by yuhan** [comment]
* **#24330** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24330?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#24391](https://github.com/dagster-io/dagster/pull/24391) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24391?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#24404](https://github.com/dagster-io/dagster/pull/24404) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24404?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yuhan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by yuhan** [comment]
* **#24330** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24330?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#24391](https://github.com/dagster-io/dagster/pull/24391) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24391?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#24404](https://github.com/dagster-io/dagster/pull/24404) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24404?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yuhan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Pull request merged**